pkg/src/src/main/scala/edu/berkeley/cs/amplab/sparkr/SparkRBackendHandler.scala

```diff
@@ -160,6 +166,7 @@ class SparkRBackendHandler(server: SparkRBackend) extends SimpleChannelInboundHa
         }
       }
       if (!parameterWrapperType.isInstance(args(i))) {
+        logInfo(s"arg $i not match: type $parameterWrapperType != ${args(i).getClass()}")
```

Comment:
This is very useful to have. Could we add a little bit more detail here to say what is expected and what was provided ?
i.e. Something like 
`logInfo(s"arg $i does not match: Expected type $parameterWrapperType, but got ${args(i).getClass()}")`